optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/aggregator/swingui/BenchmarkAggregatorFrame.java

```diff
@@ -84,6 +85,7 @@
     /**
      * Reads an XML benchmark configuration from the classpath
      * and uses that {@link PlannerBenchmarkConfig} to do an aggregation.
+     * 
```

Comment:
Trailing space.

Reply:
OK, this seems to be done by the Eclipse formatter. There's nothing we can do about it, I guess.

Reply:
Nice catch. This is truly done by the formatter - both the IDEA and maven plugin. If there is already an empty line before the `@param` mark, no space is added. However, when the formatter needs to include the empty line, it does include the trailing space as well.

Unfortunately, I don't see any systematic way of avoiding this. I quickly checked the Quarkus codebase and it has the same issue too.

Reply:
I checked again and crossed out my previous comment. After enabling Eclipse formatter plugin and formatting the code in my IDEA, empty Javadoc lines were inserted but without the trailing space.

It could be because I have this enabled:
![Settings_005](https://user-images.githubusercontent.com/673386/79206623-e4823780-7e3f-11ea-8d49-adc1da2334d6.png)

I recommend this setting to everyone.

Reply:
+1 for using this setting. On the other hand, it does not prevent such a trailing space from occurring. The maven formatter plugin adds them too and so does Eclipse IDE.